readme.md

```diff
@@ -60,6 +60,12 @@ Directory for data files.
 
 Directory for config files.
 
+Example locations (with the default `nodejs` [suffix](#suffix)):
+
+- macOS: `~/Library/Preferences/MyApp-nodejs`
+- Windows: `%APPDATA%\MyApp-nodejs\Config`
```

Comment:
Would be useful to also show an example of the full resolved path.

Reply:
My rationale for `%APPDATA%` is that it's better / easier to read than `C:\Users\sindresorhus\AppData\Roaming` (and even that depends on the Windows version I believe).

While we're at it, do you think `~` is fine? I prefer `~/Library/Preferences/MyApp-nodejs` over `/Users/sindresorhus/Library/Preferences/MyApp-nodejs` but it's a similar issue as with `%APPDATA%`. Overall, my preference is towards shorter, more generic examples where possible.

Reply:
> My rationale for %APPDATA% is that it's better / easier to read

I agree. I meant just showing an example of the resolved path afterwards.

> While we're at it, do you think ~ is fine?

Yes

Reply:
Oh, so what do you mean by "resolved path"? `paths.config` is a directory so examples like `~/Library/Preferences/MyApp-nodejs` or `%APPDATA%\MyApp-nodejs\Config` are already fully resolved, aren't they?

Reply:
From experience, people know what `~` resolves to, but it's not always clear what `%APPDATA` resolves to.

Reply:
Could you show an example of what you'd like to see? I'm afraid I don't fully understand 😄.

Current text:

```txt
- macOS: `~/Library/Preferences/MyApp-nodejs`
- Windows: `%APPDATA%\MyApp-nodejs\Config`
- Linux: `~/.config/MyApp-nodejs` (or `$XDG_CONFIG_HOME/MyApp-nodejs`)
```

You'd like to see:

???

Thanks.

Reply:
```
- macOS: `~/Library/Preferences/MyApp-nodejs`
- Windows: `%APPDATA%\MyApp-nodejs\Config` (Example: `C:\Users\USERNAME\AppData\Roaming\MyApp-nodejs\Config`)
- Linux: `~/.config/MyApp-nodejs` (or `$XDG_CONFIG_HOME/MyApp-nodejs`)
```

Reply:
Thanks. I've changed the format and also documented the other properties, see [`readme.md`](https://github.com/sindresorhus/env-paths/blob/f4f8c051dc76b04ad1707b387f9c0b3e73dc8356/readme.md) as of f4f8c051d.

It would be great if you could double-check the actual values, for example, I'm not that sure that `.temp` examples are fully correct on Windows & Linux. But if you could review the other as well that would be great 🙏.

After this is checked, I'll be happy to update the `.d.ts` file as well.

Reply:
> for example, I'm not that sure that .temp examples are fully correct on Windows & Linux

It's all in the code. I don't have a Linux or Windows system to test on right now. The Linux one for temp looks incorrect from a quick look as (per the code) it should include the username.

Reply:
I tried to work with the source code closely, for all the examples, I hope that `.temp` is the only one I missed – you're right, it should contain username. For example, https://runkit.com/embed/kefcnj8b9f7d.

Fixed in ff5b24a8745d4b0f4a9ddfa8c4cc1e72ab1cd9e7.

Reply:
> You also need to update the docs in `index.d.ts`

Done in 2fff90a.